theme/oauth-authorize.ftl

```diff
@@ -0,0 +1,111 @@
+[#ftl/]
+[#setting url_escaping_charset="UTF-8"]
+[#-- @ftlvariable name="application" type="io.fusionauth.domain.Application" --]
+[#-- @ftlvariable name="client_id" type="java.lang.String" --]
+[#-- @ftlvariable name="code_challenge" type="java.lang.String" --]
+[#-- @ftlvariable name="code_challenge_method" type="java.lang.String" --]
+[#-- @ftlvariable name="devicePendingIdPLink" type="io.fusionauth.domain.provider.PendingIdPLink" --]
+[#-- @ftlvariable name="hasDomainBasedIdentityProviders" type="boolean" --]
+[#-- @ftlvariable name="identityProviders" type="java.util.Map<java.lang.String, java.util.List<io.fusionauth.domain.provider.BaseIdentityProvider<?>>>" --]
+[#-- @ftlvariable name="loginId" type="java.lang.String" --]
+[#-- @ftlvariable name="metaData" type="io.fusionauth.domain.jwt.RefreshToken.MetaData" --]
+[#-- @ftlvariable name="nonce" type="java.lang.String" --]
+[#-- @ftlvariable name="passwordlessEnabled" type="boolean" --]
+[#-- @ftlvariable name="pendingIdPLink" type="io.fusionauth.domain.provider.PendingIdPLink" --]
+[#-- @ftlvariable name="redirect_uri" type="java.lang.String" --]
+[#-- @ftlvariable name="rememberDevice" type="boolean" --]
+[#-- @ftlvariable name="response_type" type="java.lang.String" --]
+[#-- @ftlvariable name="scope" type="java.lang.String" --]
+[#-- @ftlvariable name="showCaptcha" type="boolean" --]
+[#-- @ftlvariable name="showPasswordField" type="boolean" --]
+[#-- @ftlvariable name="state" type="java.lang.String" --]
+[#-- @ftlvariable name="tenant" type="io.fusionauth.domain.Tenant" --]
+[#-- @ftlvariable name="tenantId" type="java.util.UUID" --]
+[#-- @ftlvariable name="timezone" type="java.lang.String" --]
+[#-- @ftlvariable name="user_code" type="java.lang.String" --]
+[#-- @ftlvariable name="version" type="java.lang.String" --]
+[#import "../_helpers.ftl" as helpers/]
+[#assign PERM_DOMAIN="{ENTER DOMAIN HERE}"]
+
+<!doctype html>
+<html lang="en">
+    [@helpers.permHeadTag title="Reset Password"]
```

Comment:
```suggestion
    [@helpers.permHeadTag title="Authorize Application"]
```

Reply:
Or actually maybe this page should just say "Log in to Permanent.org", as our existing login page does? or more specifically, the page text says "Log in to Permanent.org", while the title says "Log In | Permanent.org".

Reply:
Yep, realizing I just copy/pasted and forgot to change the value! I think I can also use something like the following for both the page text and title text:

```
[@helpers.permHeadTag title=theme.message('login')]
```

Reply:
Oh, yes, that's much better!